module3/Math.rb

```diff
@@ -0,0 +1,74 @@
+# Demo for writing class and class methods
+
+
+class  Number
+  def self.addition(*numbers)           #method for additin of numbers accepting variable number of arguments
+    iterator = 0
+    sum = 0
+    while iterator < numbers.length
+      sum += numbers[iterator]          #adding all input numbers
+      iterator += 1
+    end
+    sum                                 #last statement is return value              
+  end
+
+  def self.multiplication(*numbers)     #method for multiplication of numbers  
+    iterator = 0
+    product = 1
+    while iterator < numbers.length
+      product *= numbers[iterator]
+      iterator += 1
+    end
+    product
+  end
+
+  def self.substraction(number1, number2) #method for substraction of numbers
+    number1 - number2
+  end
+
+  def self.division(number1, number2)     #method for division of numbers
+    number1 / number2
+  end
+
+  def self.is_prime(number)               #method for if number is prime
+    flag = true
+    iterator = 2
+    if number < 2
+      flag = false
+    elsif number > 2
+      while iterator <= number/2
```

Comment:
Simply you can write this method as:
def self.is_prime(number) 
  (2..(number-1)).each do |iterator|
    return false if (number % iterator == 0)
  end
  true
end